README.md

```diff
@@ -1,2 +1,83 @@
 # Reps-Mobile
-reps.mozilla.org data on mobile
+
+<p align="center">
+  <a href="https://flutter.dev/">
+    <img alt="Flutter" src="https://flutter.dev/assets/flutter-lockup-4cb0ee072ab312e59784d9fbf4fb7ad42688a7fdaea1270ccf6bbf4f34b7e03f.svg" width="170" />
```

Comment:
This repository is for Reps and not for Flutter... If you are adding a logo I would suggest you use Reps logo.

Reply:
Can we use that from the one from the Reps Website? 
Will there be any Copyright issue?

Reply:
Please find the original files for the logo here: https://drive.google.com/drive/folders/1M80zSrx640Ve7chhrmwn08RO0s6HnN5n

Reply:
ok I'll look for the same

Reply:
Done!